cmd/interrupt.go

```diff
@@ -0,0 +1,28 @@
+// +build !windows
+
+package cmd
+
+import (
+	"os"
+	"os/exec"
+	"syscall"
+)
+
+// interruptProcess sends an Interrupt signal to the process.
+func interruptProcess(proc *os.Process) error {
+	return proc.Signal(os.Interrupt)
```

Comment:
If you sent SIGTERM bash wouldn't catch it in the test. I think SIGTERM makes more sense if we consider that a dapr app is likely to be a daemon...